internal/resolve/catalog.go

```diff
@@ -158,6 +179,7 @@ func (r *CatalogResolver) Resolve(ctx context.Context, ext *ocv1.ClusterExtensio
 
 	// Check for ambiguity
 	if len(resolvedBundles) != 1 {
+		l.Info("resolution failed", "stats", catStats)
```

Comment:
Do we want this before all returns from this point forward?

Reply:
We could? I felt like those other returns include context in the error message that would make it possible to figure out what happened, so that's why I only included it here.